### PR TITLE
Use monitor-dog. add sockets reporting

### PR DESCRIPTION
**Pull request by podviaznikov**
Use monitor dog module.
This enables reporting of open/pending sockets and file descriptors.

### CODE REVIEW

**Comment by podviaznikov** [comment]
can someone review this?

File: main.js [hunk 1]
```diff
@@ -12,11 +12,17 @@ var app = require('./lib/app.js');
...
+var noop = require('101/noop');
+
+
 module.exports = {
   start: start,
   stop: stop
 };
 
+
+
```
Comment (bkendall) [inline main.js]: such whitespace...

File: main.js [hunk 2]
```diff
@@ -30,6 +36,7 @@ function start (port, cb) {
     if (err) { return cb(err); }
     debug('server listen on', port);
     listener.start(publisher, cb);
+    monitor.startSocketsMonitor();
```
Comment (bkendall) [inline main.js:35]: does this need to start before the listener starts? the ordering just looks funny to me...

**Comment by bkendall** [comment]
LGTM

**Pull request merged**